lib/iris/fileformats/grib/__init__.py

```diff
@@ -962,7 +970,7 @@ def as_pairs(cube):
 def as_messages(cube):
     """
     Convert one or more cubes to GRIB messages.
-    Returns an iterable of grib_api GRIB messages.
+    Returns an iterable of grib_api GRIB message IDs.
```

Comment:
Actually, there is some serious potential for confusion here, when we make 'message.GribMessage' public, about the fact that 'as_pairs' and 'as_messages' do **_not_** handle those, but grib_api handles instead.
I.E. despite losing 'GribWrapper', we still have a low-level and a higher-level message concept.

We could consider renaming 'GribMessage' to something that made it clearer that this is **_not_** the things that the grib_api deals in ?  I can now see that was the point of the name 'GribWrapper' !